docs/Recommendations.md

```diff
@@ -8,21 +8,20 @@ Fastify.
 * [Use A Reverse Proxy](#reverseproxy)
 * [Kubernetes](#kubernetes)
 
-## Use A Reverse Proxy
+## Use a Reverse Proxy
```

Comment:
It seems to me this, and in the PR on errors, is an attempt to enforce https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case

My view is that we ignore those rules and either capitalize every word in a heading or capitalize only the first word. By trying to introduce any other syntax rules we just get into a mess like is described in https://www.chicagomanualofstyle.org/qanda/data/faq/topics/CapitalizationTitles/faq0007.html

My preference is to capitalize every word in a heading.

Reply:
There doesn't seem to be a consistent style for headings/titles across the documentation, i.e. the [Write-Plugin](https://github.com/fastify/fastify/blob/master/docs/Write-Plugin.md) file has a mix of lower and uppercase.

Happy to amend titles/headers to be consistent.

Reply:
Agreed that it isn't consistent across all of the documents. That is something we should address.